docker-compose.yml

```diff
@@ -93,6 +93,21 @@ services:
     environment:
       - SERVER_PORT=8080
       - OAUTH_ENDPOINT_URL=http://hmpps-auth:8090/auth
+      - ASSESSMENTAPI_BASEURL=http://offender-assessments-api:8080
```

Comment:
transitive dependencies? ouch

Reply:
sadly, we have this already all over the place. e.g. hmpps auth requires community api to allow delius logins.

Reply:
yeah, but if this keeps up, we won't be able to create a working dev env or start getting circular dependencies... need to take this forward with tooling

Reply:
agree